src/index.js

```diff
@@ -5,107 +5,209 @@ import './view/css/style.css';
-    this.date = newDate;
-  }
-
-  const getDate = () => {
-    return this.date;
-  }
-
-  const changeChecked = () => {
-    this.checked = !this.checked;
-  }
-
-  const getChecked = () => {
-    return this.checked;
-  }
-
-  return {
-    setName,
-    setDescription,
-    setDate,
-    changeChecked,
-    getName,
-    getDescription,
-    getDate,
-    getChecked
-  };
+function editTodo(todoIndex) {
+  const newName = document.getElementById('edit-todo-title').value;
+  projectList[currentProjectIndex].todos[todoIndex].title = newName;
+
+  const newDate = document.getElementById('edit-todo-dueDate').value;
+  projectList[currentProjectIndex].todos[todoIndex].dueDate = newDate;
+
+  const newPrior = document.getElementById('edit-todo-priority').value;
+  projectList[currentProjectIndex].todos[todoIndex].priority = newPrior;
+
+  const newDesc = document.getElementById('edit-todo-description').value;
+  projectList[currentProjectIndex].todos[todoIndex].description = newDesc;
+
+  saveLocal();
+  renderTodos(projectList[currentProjectIndex]); // eslint-disable-line no-use-before-define
 }
 
-const render = () => {
+function setTodoModal(todoIndex) {
+  document.getElementById('edit-todo-title').value = projectList[currentProjectIndex].todos[todoIndex].title;
+  document.getElementById('edit-todo-dueDate').value = projectList[currentProjectIndex].todos[todoIndex].dueDate;
+  document.getElementById('edit-todo-priority').value = projectList[currentProjectIndex].todos[todoIndex].priority;
+  document.getElementById('edit-todo-description').value = projectList[currentProjectIndex].todos[todoIndex].description;
+}
 
+// add todo to project save local and render
+function addTodo(todoList) {
+  const todoTitle = document.getElementById('new-todo-title').value;
+  const todoDueDate = document.getElementById('new-todo-dueDate').value;
+  const todoPriority = document.getElementById('new-todo-priority').value;
+  const dotoDescription = document.getElementById('new-todo-description').value;
+  const todo = new Todo(todoTitle, todoDueDate, todoPriority, dotoDescription);
+  todoList.push(todo);
+
+  saveLocal();
+  location.reload(); // eslint-disable-line no-restricted-globals
 }
 
-const saveLocalAndRender = () => {
-  localStorage.setItem('LocalTaskArray', JSON.stringify(myLibrary));
-  render();
+// add button actions
+const btnAddProject = document.getElementById('addProject');
+btnAddProject.onclick = () => { addProject(); };
+
+const btnAddTodo = document.getElementById('addTodo');
+btnAddTodo.onclick = () => { addTodo(projectList[currentProjectIndex].todos); };
+
+const btnEditProject = document.getElementById('editProject');
+btnEditProject.onclick = () => { editProject(btnEditProject.getAttribute('curr-project-index')); };
+
+const btnEditTodo = document.getElementById('editTodo');
+btnEditTodo.onclick = () => { editTodo(btnEditTodo.getAttribute('curr-todo-index')); };
+
+// render todos table
+function renderTodos(project) {
```

Comment:
You ought to use ES6 Syntax for this project:
Which means usage of arrow functions instead of the function keyword.
See this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/es6_requirement.png) from pathwright